dallinger/command_line.py

```diff
@@ -296,26 +296,11 @@ def debug(verbose):
 
     # Try opening the psiTurk shell.
     try:
-        p = pexpect.spawn("psiturk")
-        p.expect_exact("]$")
-        p.sendline("server on")
-        p.expect_exact("Experiment server launching...")
-
-        # Launche the experiment.
-        time.sleep(4)
-
-        host = config.get("Server Parameters", "host")
-        port = config.get("Server Parameters", "port")
-
-        subprocess.call(
-            'curl --data "" http://{}:{}/launch'.format(host, port),
-            shell=True)
-
-        log("Here's the psiTurk shell...")
+        p = pexpect.spawn("python", [os.path.realpath(os.path.join(__file__, '..', 'heroku', 'psiturkapp.py'))])
+        log("Here's the gunicorn app...")
         p.interact()
-
     except Exception:
```

Comment:
I would like to either get rid of the bare `except` statement or make sure that we log the original traceback. Otherwise we're hiding valuable information about what went wrong.